Content.Shared/Forensics/ForensicScannerEvent.cs

```diff
@@ -0,0 +1,23 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Forensics
+{
+    [Serializable, NetSerializable]
+    public sealed class ForensicScannerUserMessage : BoundUserInterfaceMessage
+    {
+        public readonly List<string> Fingerprints = new();
+        public readonly List<string> Fibers = new();
```

Comment:
do fingerprints and fibers need to be different necessarily? i dont see why you couldn't just reuse the same component for them and treat them the same (this is more of a high level question than just these 2 lines but im realizing it now)